src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,62 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+
+    private Random random = new Random();
+
+    public String getRandomFigure(int figureNumber) {
```

Comment:
```suggestion
    public Figure getRandomFigure(int figureNumber) {
```
цей метод повинен повертати фігуру, з нею значно простіше працювати чим з строкою